scripts/reporting/get_all_timestamps.py

```diff
@@ -0,0 +1,27 @@
+'''
+Gets timestamps for the REDCap All Forms Dashboard
+'''
+import sibispy
+import sys
+
+'''
+All arms necessary for getting all the dates
+'''
+arms = ['18month_followup_arm_1', '4y_visit_arm_1', '78month_followup_arm_1', '1y_visit_arm_1', '54month_followup_arm_1', '7y_visit_arm_1', '2y_visit_arm_1', '5y_visit_arm_1', '8y_visit_arm_1', '30month_followup_arm_1', '66month_followup_arm_1', '90month_followup_arm_1', '3y_visit_arm_1', '6month_followup_arm_1', 'baseline_visit_arm_1', '42month_followup_arm_1', '6y_visit_arm_1']
```

Comment:
You should be able to just leave `events` unspecified in the call on L25 and that should net you all event visit dates for free, without the need to explicitly list them here like this.